bundles/org.openhab.binding.hue/src/main/resources/OH-INF/i18n/hue.properties

```diff
@@ -182,3 +273,14 @@ actionDesc = Send a light command with a custom fade time.
 # discovery results
 
 discovery.group.all_lights.label = All lights
+
+# api v2 dynamic actions
+
+dynamics.action.label = Send a dynamic command.
+dynamics.action.description = Send a dynamic command to a device, room or zone.
```

Comment:
The convention is "Sends a ..."

Reply:
> convention is "Sends a ..."

With first letter capitalised? With a period at the end?

Reply:
Yes. Just replace "Send" by "Sends".